source/manuals/grafana-dashboards.html.md.erb

```diff
@@ -1,58 +1,61 @@
 ---
-title: Viewing your metrics using Grafana dashboards
+title: View and create Grafana dashboards
 ---
 
 # <%= current_page.data.title %>
 
+*there should be a brief intro here - what is this content about, who’s it for?*
 
-## Logging into Grafana
+## Access Grafana
 
-Visit [grafana-paas.cloudapps.digital](https://grafana-paas.cloudapps.digital) and sign in with Google.
+Sign in to Grafana at [grafana-paas.cloudapps.digital](https://grafana-paas.cloudapps.digital) using your GDS Google account.
 
 ![google-auth](images/google-auth.png)
 
+## Display dashboards
 
-## Viewing your dashboards
+Team dashboards are organised into separate Grafana folders.
 
-Dashboards for teams have been organised into different Grafana folders.
+Select **Home** in the top left of the Grafana screen to choose your team’s dashboard folder and view your dashboards.
 
-Click 'Home' in the top left of the screen to pick your team's dashboard folder and see your dashboards.
+### Display using your team TV
 
+You should use your TV’s Google Chromebit user to display your Grafana dashboard. The Google Chromebit user only has read access to Grafana.
 
-## Creating and editing dashboards
+You should not use your personal Google account to display your Grafana dashboard on your team’s TV. Your personal Google account may have editing or admin permissions.
 
-In order to create or edit a dashboard in your team folder you will need to have the relevant permissions. Admin/editor permissions for your team folder can be granted by contacting us on the `#re-prometheus-support` Slack channel.
+## Create and edit dashboards
 
-If you don't have a team folder, contact us on the `#re-prometheus-support` Slack channel to create the folder and get admin rights to manage it. If you would like your own copy of any of the default dashboards that you can personalise then please ask for this too.
+Contact Reliability Engineering using the [#re-prometheus-support Slack channel][https://gds.slack.com/messages/CAF5H4N4Q/#] to request:
 
+* admin permissions to create or edit a dashboard in your team folder
+* a new team folder and the admin rights to manage it
```

Comment:
Is there a naming convention people should follow?

Reply:
It's still kind of early days with the team folders so there aren't any super precise rules at the moment